app/src/organisms/ODD/QuickTransferFlow/Aspirate/hooks/useAspirateSettingsConfig.ts

```diff
@@ -54,13 +54,20 @@ export function useAspirateSettingsConfig({
         setSelectedSetting(SETTING_OPTIONS.ASPIRATE_TIP_POSITION)
       },
     },
-    // ToDo replace dummy configs for submerge
     {
       option: SETTING_OPTIONS.ASPIRATE_SUBMERGE,
       copy: t('submerge'),
-      value: '', // t('submerge_value', { volume: 'dummy' }),
-      enabled: false,
-      onClick: () => {},
+      value:
+        state.submergeAspirate !== undefined
+          ? t('submerge_value', {
+              speed: state.submergeAspirate?.speed,
+              position: state.submergeAspirate?.positionFromBottom,
```

Comment:
nit, you're already checking if they're undefined in line 61
```suggestion
              speed: state.submergeAspirate.speed,
              position: state.submergeAspirate.positionFromBottom,
```